core/src/main/java/io/neow3j/protocol/core/Neow3jExpress.java

```diff
@@ -0,0 +1,158 @@
+package io.neow3j.protocol.core;
```

Comment:
I think this class should reside next to `Neow3j` in the package `io.neow3j.protocol`.

Reply:
agree